README.md

```diff
@@ -269,6 +269,14 @@ profile.registerType(classToRegister, locationToRegister, 'container', 'listed')
     // querying the registry now will include the new container
     profile.typeRegistryForClass(vocab.sioc('Post'))
   })
+
+// To remove the same class from registry:
+var classToRemove = vocab.sioc('Post')
+profile.unregisterType(classToRemove, 'listed')
```

Comment:
I wonder if can maybe just use a boolean instead of `listed` and `unlisted`. Set it to `true` if the registry should be public, otherwise it's `false` by default.

```
profile.unregisterType(classToRemove, true)
```